main.py

```diff
@@ -10,34 +10,98 @@ def main():
     """Função principal que será rodada quando o script for passado para o interpretador."""
 
     # A função abaixo abre o arquivo texto numeros.txt em modo leitura e lê as linhas dele separando
-    # elas em uma lista de strings.
-    with open('números.txt', 'r', encoding='utf8') as arquivo:
-        linhas_do_arquivo = arquivo.readlines()
+    # elas em uma .
+    with open('números.txt','r', encoding='utf8') as arquivo:
+       linhas_do_arquivo = arquivo.readlines()
+
+
+
 
     # Pegue a lista de strings e converta todos os valores dentro dela para inteiros.
     # TODO COLOQUE SEU CÓDIGO AQUI E APAGUE ESSE COMENTÁRIO DEPOIS.
+def lista_ordenada_bubble_sort(items):
+  had_swap = True
+  while had_swap:
+    had_swap = False
+    for i in range(len(items) - 1):
+      if items[i] > items[i + 1]:
+        swap(items, i)
+        had_swap = True
+        return items
+def swap(items, index):
+ items[index], items[index + 1] = items[index + 1], items[index]       
+
+
+def lista_ordena_mergeSort(array):
+    if len(array) > 1:
+      r = len(array)//2
+      L = array[:r]
+      M = array[r:]
+      mergeSort(L)
+      mergeSort(M)
+
+      i = j = k = 0
+      while i < len(L) and j < len(M):
+            if L[i] < M[j]:
+                array[k] = L[i]
+                i += 1
+            else:
+                array[k] = M[j]
+                j += 1
+            k += 1
+      while i < len(L):
+            array[k] = L[i]
+            i += 1
+            k += 1
+
+      while j < len(M):
+            array[k] = M[j]
+            j += 1
+            k += 1
+
+def lista_ordena_selectionSort(array, size):
+   
+    for step in range(size):
+        min_idx = step
+
+        for i in range(step + 1, size):
+          if array[i] < array[min_idx]:
+                min_idx = i
+                (array[step], array[min_idx]) = (array[min_idx], array[step])
+
+
+data = [-2, 45, 0, 11, -9]
+size = len(data)
+selectionSort(data, size)
```

Comment:
Aqui está fora da identanção, acho que queriam colocar no lista_ordena_selectionSort?
```suggestion
    data = [-2, 45, 0, 11, -9]
    size = len(data)
    selectionSort(data, size)
```